### ISSUE TITLE
The code annotation (about 'enable_http_compression') is wrong. 

### ISSUE THREAD

**Issue by ybkim04** [ClickHouse/clickhouse-js#134]
The default value of option 'enable_http_compression' is not false

```
    /** Compress the result if the client over HTTP said that it understands data compressed by gzip or deflate. (default: false) */
    enable_http_compression?: Bool;
```

Unlike the option default value specified as false when creating a client that inquires DB with a Readonly user,
The error below is exposed.
`Cannot modify 'enable_http_compression' setting in readonly mode.`

REF: https://github.com/ClickHouse/clickhouse-js/blob/97a61622b5d979f2326061845735d1d01472ad04/src/settings.ts#L317

**Comment by slvrtrn** [comment]
Thanks for pointing it out.

That one we parsed from the server-side C++ code of ClickHouse itself (for example [this](https://github.com/ClickHouse/ClickHouse/blob/46ed4f6cdf68fbbdc59fbe0f0bfa9a361cc0dec1/src/Core/Settings.h#L44))

The reason why we added these additional typings is that you could provide these settings on a query basis ([for example](https://github.com/ClickHouse/clickhouse-js/blob/97a61622b5d979f2326061845735d1d01472ad04/__tests__/integration/clickhouse_settings.test.ts#L21-L24)), so it is better to have some types. I guess we should remove any mention of `default` there, as the client actually has it `true` [for requests by default](https://github.com/ClickHouse/clickhouse-js/blob/main/src/client.ts#L26-L31).

**Comment by slvrtrn** [comment]
Resolved by #193
